docs/administration-stats.md

```diff
@@ -22,15 +22,15 @@ All metrics below are *reset* to 0 upon broker restart, *except* gauges marked a
 | msgRateOut                                       | The sum of all local and replication consumers' dispatch rates (message per second).                                            |
 | msgThroughputOut                                 | The sum of all local and replication consumers' dispatch rates (byte per second).                                               |
 | averageMsgSize                                   | The average size (bytes) of messages published within the last interval.                                                        |
-| storageSize                                      | The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic. Not reset upon restart |
-| earliestMsgPublishTimeInBacklogs                 | The publish time of the earliest message in the backlog (in milliseconds). Not reset upon restart                               |
+| storageSize*                                      | The sum of the ledgers' storage size **in BookKeeper** for a topic (in bytes). <br/><br/>**Note**: the `total storage size of a topic` = `storageSize` + `offloadedStorageSize`.  |
```

Comment:
Do we need to add a code sign for "total storage size of a topic"? Same comment on the next one.

Reply:
What does "code sign" refer to? Do you mean adding backslashes?